templates/CRM/Admin/Form/MessageTemplates.tpl

```diff
@@ -124,3 +147,41 @@
 </div>
 </div> <!-- end of crm-form-block -->
 {include file="CRM/Mailing/Form/InsertTokens.tpl"}
+
+{literal}
+  <script type='text/javascript'>
+    CRM.$(function($) {
+      var mimeType = null;
+      // if default file is selected then hide the file upload field
+      if ($('a.delete-attachment').length) {
+        $('#file_id').hide();
+        mimeType = $('a.delete-attachment').data('mimetype');
+      }
+
+      $('#file_type').on('change', function(){
+        toggleByFileType(this.value);
+      });
+      toggleByFileType($('#file_type').val());
+
+      function toggleByFileType(type) {
+        var show = (type == 'odt' || type == 'docx') ? false : true;
+        $("#msg_html").toggle(show);
+        $("#msg_text").toggle(show);
```

Comment:
Trust me I tried to join the selectors but don't knw why this leads to distorted HTML accordion.